### ISSUE TITLE
Accessibility (JAWs) scroll not working correctly when traversing with Arrow Keys on FilterPanel

### ISSUE THREAD

**Issue by Pasoon** [carbon-design-system/ibm-security#913]
### Description

This issue was found with the FilerPanel component. When using the arrow keys with JAWs running the scroll inside of the Search Filter does not seem to work correctly, JAWs begins to read all the items that are not visible to user. BUT if you tab the scrolling works correctly. A team logged this Accessibility error for us letting us know that when using arrow keys it does not scroll when jaws is being used. In other components like our Table for example the scroll works fine when using arrow keys. Below is a video of the issue. 

https://user-images.githubusercontent.com/17908476/111226612-c412ae00-85b7-11eb-9c8b-4afb712ee6fb.mp4

#### Expected behavior

When using arrow keys with JAWs the filter search drop down should scroll down when trying to access items and view items underneath. 

#### Actual behavior

The drop down does not scroll down, JAWs continues to read items that are not visible or within scrolled content

### Steps for reproducing

- Login to QRadar
- Click Use Case Manager tab
- using the JAWS 2021 read this page
- Clicked the Find filter search bar at the top left
- Begin traversing with arrow keys downwards

### Affected browsers

- [Chrome 88 on macOS 11.2, production (QRadar VM) ]
